slack_bolt/slack_function/slack_function.py

```diff
@@ -46,21 +46,20 @@ def action(
     ) -> Callable[..., Optional[Callable[..., Optional[BoltResponse]]]]:
         """Registers a new action listener to your function. This method can be used as either a decorator or a method.
 
-            # Use this method as a decorator
+            # Define a function handler using the function decorator
             @app.function("request-approval")
             def request_approval(event, complete: Complete):
-                complete(outputs={})
+                // do something
```

Comment:
// line comment does not work in Python
```suggestion
                # do something
```

Reply:
Seems like my java reflexes kicked in here 😄